docs/cpu-target.md

```diff
@@ -264,6 +279,7 @@ TODO
 
 # Main
 
+* groupshared is not yet supported
```

Comment:
I would add barriers here as a missing feature, since they are closely related to `groupshared`.